client-side/src/signUp/signUp.scss

```diff
@@ -0,0 +1,61 @@
+
+/* src/SignUp.css */
+.signup-container {
+    max-width: 400px;
+ 
+    margin: 0 auto;
+    padding: 20px;
+    border: 1px solid #ccc;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
```

Comment:
in all your styles code please use % instead of px's and colors and fonts should come from variables